.github/workflows/publish-release.yml

```diff
@@ -17,7 +23,10 @@ on:
 jobs:
   release:
     name: Release
-    uses: OpenDevicePartnership/patina-devops/.github/workflows/ReleaseWorkflow.yml@v0.2.2
+    uses: OpenDevicePartnership/patina-devops/.github/workflows/ReleaseWorkflow.yml@v0.3.0
     with:
       ignore-prefix: 'patina-v'
     secrets: inherit
+    permissions:
+      contents: write
```

Comment:
todo in another PR & sync but these permissions were removed for this workflow. They are getting added back here and should not be.